### ISSUE TITLE
No need to `stats.registerView(view)`

### ISSUE THREAD

**Issue by DazWilkin** [census-instrumentation/opencensus-node#287]
Please answer these questions before submitting a bug report.

### What version of OpenCensus are you using?
```
0.0.8
```
### What version of Node are you using?
```
node --version
v10.13.0
```
### What did you do?
If possible, provide a recipe for reproducing the error.

Attempting to run `examples/stackdriver.js`

### What did you expect to see?

Metrics should not be exported unless they are captured by Views.

In the example, although the views are created, they're not registered, yet the sample works.

This works but it ought not:
```
// Register the view.
const latencyView = stats.createView(
  "demo/latency",
  mLatencyMs,
  AggregationType.DISTRIBUTION,
  [tagKey],
  "The distribution of the repl latencies",
  // Latency in buckets:
  // [>=0ms, >=25ms, >=50ms, >=75ms, >=100ms, >=200ms, >=400ms, >=600ms, >=800ms, >=1s, >=2s, >=4s, >=6s]
  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
);
```

### What did you see instead?

The code should (IIUC) look like this:

```
// Create & Register the view.
const latencyView = stats.createView(
  "demo/latency",
  mLatencyMs,
  AggregationType.DISTRIBUTION,
  [tagKey],
  "The distribution of the repl latencies",
  // Latency in buckets:
  // [>=0ms, >=25ms, >=50ms, >=75ms, >=100ms, >=200ms, >=400ms, >=600ms, >=800ms, >=1s, >=2s, >=4s, >=6s]
  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
);
stats.registerView(latencyView);
```

### Additional context

With the existing code, after the `stats.createView`, `stats.registeredViews` creates a view (but I think it should not [yet]) called `repl/latency`, the view is named after the measure.

**Comment by mayurkale22** [comment]
Currently, ```createView()``` method is combined with ```registerView()``` (internal call). I have opened an issue to deperecate this behavior. As you mentioned, ideally user should call ```registerView()``` explicitly to listen to new measurements in its measure.

**Comment by mayurkale22** [comment]
The fix will be available in the new version (0.0.9), closing this one. @DazWilkin thanks for the issue.
